api/server/structs/conversions_lightclient.go

```diff
@@ -4,150 +4,48 @@ import (
 	"encoding/json"
 )
 
+func (h *LightClientHeader) ToRawMessage() (json.RawMessage, error) {
+	return json.Marshal(h)
+}
+
+func (h *LightClientHeaderCapella) ToRawMessage() (json.RawMessage, error) {
+	return json.Marshal(h)
+}
+
+func (h *LightClientHeaderDeneb) ToRawMessage() (json.RawMessage, error) {
+	return json.Marshal(h)
+}
+
 func LightClientBootstrapResponseFromJson(data []byte) (*LightClientBootstrapResponse, error) {
```

Comment:
are these functions needed? I think inlining `unmarshal` will make the code more readable